APIMatic.Core.Test/Pagination/Paginator.cs

```diff
@@ -0,0 +1,191 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using System.Linq;
+using System.Net.Http;
+using APIMatic.Core.Pagination.Strategies;
+using APIMatic.Core.Test.Api;
+using APIMatic.Core.Test.MockTypes.Models.Pagination;
+using APIMatic.Core.Test.MockTypes.Pagination;
+using APIMatic.Core.Test.MockTypes.Pagination.MetaData;
+using APIMatic.Core.Test.Utilities;
+using NUnit.Framework;
+
+namespace APIMatic.Core.Test.Pagination
+{
+    [TestFixture]
+    public class Paginator : ApiCallTest
```

Comment:
It is conventional to use postfix `Test` for TestFixture classes, please add the postfix